src/seedu/addressbook/commands/ViewCommand.java

```diff
@@ -32,10 +34,22 @@ public CommandResult execute() {
             if (!addressBook.containsPerson(target)) {
                 return new CommandResult(Messages.MESSAGE_PERSON_NOT_IN_ADDRESSBOOK);
             }
+            System.out.println(getPrintableString(target.getName(), target.getPhone(),
+                    target.getEmail(), target.getAddress()));
             return new CommandResult(String.format(MESSAGE_VIEW_PERSON_DETAILS, target.getAsTextHidePrivate()));
         } catch (IndexOutOfBoundsException ie) {
             return new CommandResult(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    private String getPrintableString(Printable... printables) {
+        String output = "";
```

Comment:
Using String is okay but you may want to consider using StringBuilder. String is immutable in Java. When we do String manipulation such as string concatenation, Java creates a new String and discard the older String for garbage collection. Multiple concatenations will generate a lot of garbage in the heap. We avoid this by using StringBuilder as StringBuilder is a mutable class.